src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/activities/ActivitiesApiServiceTest.kt

```diff
@@ -27,7 +28,7 @@ import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.activities.model.
 import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.helper.SpringAPIServiceTest
 import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.wiremock.ActivitiesApiExtension.Companion.activitiesApi
 import java.time.LocalDate
-import java.util.UUID
+import java.util.*
```

Comment:
I thought this wasn't allowed as part of ktlint - although I have seen it go through previously

Reply:
ktlint did it! I think there are some exceptions to the no * rule and java.util is one of them.